src/scss/skin-modern/_skin-ads.scss

```diff
@@ -28,6 +28,16 @@
         }
       }
 
+      /* Let button look like it is not click-able during the skippable-in countdown */
+      &.skippable-in-countdown {
```

Comment:
What does this name mean? Why is it not part of the adskipbutton style?

Reply:
I will look for a better name and rename it.
We don't have separate files for ad related component styling.

Reply:
What about the name `countdown-active` ?

Reply:
Yap the file placement makes perfect sense.

Regarding the classname, I don't think this should be specifically taylored to the "countdown" usecase. It can just be generically called "disabled".

The beautiful solution would be to extend `Component` with an `enabled` state (similar to `hidden`) with `enable`, `disable`, `isEnabled` methods an `onEnable`, `onDisable` events. This would be quite simple to implement and make a nice feature addition that had to be added anyway at some point, and I think this is the right time now. In `_button.scss` you would add a `.disabled` style which is basically what you did here. In `AdSkipButton` you'd simply `enable`/`disable` the button accordingly.

Reply:
Added in a separate PR -> #202